lists/jo.csv

```diff
@@ -1,186 +1,187 @@
+http://www.khabarajel.com,NEWS,News Media,2014-04-15,citizenlab,
+http://www.mahjoob.com,NEWS,News Media,2014-04-15,citizenlab,
+http://www.marayanews.com,NEWS,News Media,2014-04-15,citizenlab,
+http://www.mojtma3.net,NEWS,News Media,2014-04-15,citizenlab,
+http://www.mouab.com,NEWS,News Media,2014-04-15,citizenlab,
+http://www.nabdalbalad.com,NEWS,News Media,2014-04-15,citizenlab,
+http://www.najemnewscom,NEWS,News Media,2014-04-15,citizenlab,
+http://www.nashamanews.blogspot.com,NEWS,News Media,2014-04-15,citizenlab,
+http://www.news-jo.com,NEWS,News Media,2014-04-15,citizenlab,
+http://www.nibonews.com,NEWS,News Media,2014-04-15,citizenlab,
+http://www.noornews.net,NEWS,News Media,2014-04-15,citizenlab,
+http://www.postjordan.com,NEWS,News Media,2014-04-15,citizenlab,
+http://www.qaraya.com,NEWS,News Media,2014-04-15,citizenlab,
+http://www.ramthavoice.com,NEWS,News Media,2014-04-15,citizenlab,
+http://www.rasseen.com,NEWS,News Media,2014-04-15,citizenlab,
+http://www.reefnews.info,NEWS,News Media,2014-04-15,citizenlab,
+http://www.royanews.com,NEWS,News Media,2014-04-15,citizenlab,
+http://www.rqebnews.com,NEWS,News Media,2014-04-15,citizenlab,
+http://www.s2nn.com,NEWS,News Media,2014-04-15,citizenlab,
+http://www.s2nn.com/jo,NEWS,News Media,2014-04-15,citizenlab,
+http://www.sa5arnews.com,NEWS,News Media,2014-04-15,citizenlab,
+http://www.sahafi.jo,NEWS,News Media,2014-04-15,citizenlab,
+http://www.sanews.net,NEWS,News Media,2014-04-15,citizenlab,
+http://www.sarayanews.com,NEWS,News Media,2014-04-15,citizenlab,
+http://www.sawtordon.com,NEWS,News Media,2014-04-15,citizenlab,
+http://www.secretnews.com,NEWS,News Media,2014-04-15,citizenlab,
+http://www.shaabnews.com,NEWS,News Media,2014-04-15,citizenlab,
+http://www.shayfak.net,NEWS,News Media,2014-04-15,citizenlab,
+http://www.shbabelyom.com,NEWS,News Media,2014-04-15,citizenlab,
+http://www.sheepress.com,NEWS,News Media,2014-04-15,citizenlab,
+http://www.spysatnews.com,NEWS,News Media,2014-04-15,citizenlab,
+http://www.tarbeahnews.net,NEWS,News Media,2014-04-15,citizenlab,
+http://www.templeinstitute.org,REL,Religion,2017-01-23,Alan Orth,Website of Israeli organization promoting a radical Jewish agenda for sacred sites in Jerusalem
+http://www.thejordantoday.com,NEWS,News Media,2014-04-15,citizenlab,
+http://www.tourismworld-seyaha.com,NEWS,News Media,2014-04-15,citizenlab,
+http://www.wahanews.com,NEWS,News Media,2014-04-15,citizenlab,
+http://www.wehdanews.com,NEWS,News Media,2014-04-15,citizenlab,
+http://www.wejhat.net,NEWS,News Media,2014-04-15,citizenlab,
+http://www.wtn24.com,NEWS,News Media,2014-04-15,citizenlab,
+http://www.yarmouknews.com,NEWS,News Media,2014-04-15,citizenlab,
+http://www.yawmak.com,NEWS,News Media,2014-04-15,citizenlab,
+http://www.zarqanews.net,NEWS,News Media,2014-04-15,citizenlab,
+https://www.mykalimag.com,CULTR,Culture,2018-04-12,https://storage.googleapis.com/qurium/index.html,Blocked since July 2016
```

Comment:
Missing a newline here.

Reply:
Ah I see what you did, the https://storage.googleapi.com/quirium/index.htlm is the authorname. Can you please replace the author name with your name or `anon` if you don't want to be identified, and put the url in the comment section?

Reply:
@hellais Thanks for reviewing the PR.
I made the proposed changes.